de.variantsync.core/src/main/java/de/variantsync/core/ast/LineBasedParser.java

```diff
@@ -0,0 +1,51 @@
+package de.variantsync.core.ast;
+
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class LineBasedParser {
+
+	public LineBasedParser() {}
+
+	public AST<LineGrammar, String> parseDirectory(Path folder) throws IOException {
```

Comment:
Why is the `IOException` not handled here?

Reply:
The `IOException` is not handled here in the library because otherwise it won't be shown in the IDE VariantSync is executed but only in the IDE in which VariantSync is developed.